iron-overlay-manager.html

```diff
@@ -106,6 +106,13 @@
       }
     },
 
+    getBackdropInstance: function() {
```

Comment:
I'm just wondering, if it's slightly more efficient to only create the backdrop element in case we need it? lazy-load vs eager-load ...